cmd/main.go

```diff
@@ -117,7 +117,7 @@ func main() {
 	if err != nil {
 		panic(errs.Wrap(err, "failed to create proxy"))
 	}
-	proxySrv := p.StartProxy()
+	proxySrv := p.StartProxy(proxy.ProxyPort)
```

Comment:
minor and not related to your change - I would drop `Proxy` from the name of the constant, since it's already in the name of the package. And name it something like `proxy.DefaultPort`

Reply:
thanks for your suggestion, I've renamed that